packages/did-eth-registry/.dockerignore

```diff
@@ -1,7 +1,5 @@
-.git/
```

Comment:
Some of the tooling including npm and foundry sometimes depend on git.   When I was developing in the container I noticed I couldn't run for example 'forge install' without the .git being present.

Reply:
I see.
The dev container seems to target only the `packages/did-eth-registry` folder. I guess it wouldn't include the `.git` folder from the repository root.

If this is the case, then perhaps the devcontainer should be for the whole repo?

Reply:
It seems best to offer solidity tooling in the registry and perhaps add another container later as we expand into other areas of the repo.   We can have more than one and placing a single container at the repository root would impose an unorthodox directory tree on the development workflow and integrated development tools

Reply:
In our original vision, this would not have been a problem as we were expecting to use yarn/pnpm workspaces.
This also meant that setting things up would mean running `yarn install` or `pnpm install` at the repository root.

I guess I'm just not used to containerized development yet :)

Reply:
We will continue to use the yarn install.     Also it is a best practice to try to maintain minimalistic containers.   This is something we can evolve over time according to the needs of the developers.